Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 def rocmnode(name) {
-    return 'rocmtest && miopen && (' + name + ')'
+    return '(rocmtest || miopen) && (' + name + ')'
```

Comment:
@junliume @reidkwja May I ask why this change is necessary? The latest labeling convention I am aware of is:
- Each node that allowed for use in CI is marked with "rocmtest"
- The nodes that allowed for use by MIOpen are marked with "miopen"

That is, only nodes that have both "rocmtest" and "miopen" are enabled for MIOpen CI.

Reply:
@atamazov  I was under the same impression as you but @pfultz2 informed us that `rocmtest` means shared nodes. So if we have MIOpen specific nodes we should not have `rocmtest` label on it.

Reply:
@junliume @pfultz2 So the new convention is:
- The CI nodes marked with "rocmtest" are allowed to be used in any project
- The CI nodes marked with "miopen" are only allowed for use by MIOpen
- ⚠️ From now on, setting both "rocmtest" and "miopen" labels is pointless; all such cases should be revised.

Am I right?